src/briefcase/integrations/android_sdk.py

```diff
@@ -1072,10 +1074,18 @@ def start_emulator(self, avd):
 
     https://developer.android.com/studio/run/emulator-acceleration#accel-vm
 """
-                        )
-
-                    # Try again in 2 seconds...
-                    self.sleep(2)
+                            )
+
+                        # Try again in 2 seconds...
+                        self.sleep(2)
+        except Exception:
+            # if the emulator exited, this should return its output immediately;
+            # if it is still running, this will quickly time out and print nothing.
+            with suppress(subprocess.TimeoutExpired):
+                # communicate returns (stdout, stderr); Android emulator writes
+                # INFO output to stdout and WARNING & ERROR output to stderr.
+                self.command.logger.info(emulator_popen.communicate(timeout=1)[1])
```

Comment:
Using `communicate` seemed like the best way to avoid the potential for blocking on reading from the process. Just doing `emulator_popen.stderr.read()` could block as long as the emulator is running.